ros2_ws/Dockerfile

```diff
@@ -153,3 +153,7 @@ LABEL org.opencontainers.image.description="AICA base ROS 2 image (includes ros2
 LABEL org.opencontainers.image.version="${VERSION}"
 LABEL org.opencontainers.image.base.name="${BASE_IMAGE}:${BASE_TAG}"
 LABEL tech.aica.image.metadata='{"type":"base/ws","base":{"name":"${BASE_IMAGE}:${BASE_TAG}","version":"${VERSION}"}}'
+LABEL devcontainer.metadata='[{ \
+  "containerUser": "ros2", \
+  "mounts": [ "source=${localWorkspaceFolder}/source,target=/home/ros2/ws/src,type=bind,consistency=cached" ], \
```

Comment:
Weeeell this works well for most repos, but not for dynamic components and hardware collections?

Reply:
True, although it's just a default that can always be overridden. I was mostly thinking that the user `ros2` and the workspace path `/home/ros2/ws/src` are two specific things for this image that are worth capturing in the metadata. The main intended public use case will be https://github.com/aica-technology/component-template, and other repos can overwrite as needed. Then again, a devcontainer.json configuration will always be needed to set up the image anyway so I could also drop the `mounts` specification. Let me know what you prefer.

Reply:
I lean towards removing it, it feels very specific and tailored for one certain repo/folder setup. Feels almost better if we could have those options directly in the aica-package.toml and its added as part of the development image, but then again we need a devcontainer.json file anyway so I don't know if it's worth it at all.